app/routes/meetings/components/MeetingEditor.js

```diff
@@ -60,8 +60,12 @@ function MeetingEditor({
   const [useMazemap, setUseMazemap] = useState<boolean>(false);
 
   useEffect(() => {
-    setUseMazemap(isEditPage && meeting.mazemapPoi?.value > 0);
```

Comment:
@eikhr, I just noticed we removed the `isEditPage` condition, why? 🤔 The whole reason we added the optional chaining yesterday was becuase `isEditPage` was already somehow changed to `meeting.mazemapPoi`, right?